cmake/Dependencies.cmake

```diff
@@ -1210,6 +1210,9 @@ if(USE_GLOO)
     set(__BUILD_BENCHMARK ${BUILD_BENCHMARK})
     set(BUILD_TEST OFF)
     set(BUILD_BENCHMARK OFF)
+    if(USE_ROCM)
+      set(ENV{GLOO_ROCM_ARCH} ${PYTORCH_ROCM_ARCH})
```

Comment:
You need to quote the env var value, otherwise a string such as 'gfx900;gfx906' gets truncated at the first semi-colon.

Reply:
set(ENV{GLOO_ROCM_ARCH} "${PYTORCH_ROCM_ARCH}")

Reply:
Thank you!